modules/040-node-manager/hooks/get_crds.go

```diff
@@ -443,7 +445,7 @@ func getCRDsHandler(input *go_hook.HookInput) error {
 		ngForValues["updateEpoch"] = updateEpoch
 
 		// Reset status error for current NodeGroup.
-		setNodeGroupErrorStatus(input.PatchCollector, nodeGroup.Name, "")
+		setNodeGroupStatus(input.PatchCollector, nodeGroup.Name, "error", "")
```

Comment:
NIT: "error" used in 5 places, i think it should be contstant. and "kubernetes" too.